internal/api/utils.go

```diff
@@ -265,56 +260,109 @@ func TransformComponentUnits(jsonData datatypes.JSON) map[string]interface{} {
 			if math.Abs(cpuInCores) < 1 {
 				cpuInMillicores := cpuInCores * 1000
 				cpu["amount"] = math.Round(cpuInMillicores) // millicore values are rounded & don't require decimal precision
-				cpu["format"] = "millicores"
+				cpu["format"] = "m"
 			} else {
 				cpu["amount"] = truncateToThreeDecimalPlaces(cpuInCores)
-				cpu["format"] = "cores"
+				cpu["format"] = nil
 			}
 		}
 		return nil
 	}
 
+	// Current
+	current_config, ok := data["current"].(map[string]interface{})
+	if !ok {
+		log.Error("current not found in JSON")
+	}
+
+	for _, section := range []string{"limits", "requests"} {
+
+		sectionObject, ok := current_config[section].(map[string]interface{})
+		if ok {
+			memory, ok := sectionObject["memory"].(map[string]interface{})
+			if ok {
+				err := convertMemory(memory)
+				if err != nil {
+					fmt.Printf("error converting memory in %s: %v\n", sectionObject, err)
+					continue
+				}
+			}
+			cpu, ok := sectionObject["cpu"].(map[string]interface{})
+			if ok {
+				err := convertCPU(cpu)
+				if err != nil {
+					fmt.Printf("error converting cpu in %s: %v\n", sectionObject, err)
+					continue
+				}
+			}
```

Comment:
This is maybe overkill! I just felt there could be concurrent go routines for convertMemory and convertCPU. However this is not blocker!